app/services/set_reject_by_default.rb

```diff
@@ -28,11 +28,11 @@ def call
 
 private
 
-  def beyond_eoc?(date)
+  def beyond_end_of_cycle?(date)
     date >= CycleTimetable.find_closes
```

Comment:
Should this be `CycleTimetable.reject_by_default`?

Reply:
Good spot - have made the change and added a spec